gledki_test.go

```diff
@@ -331,21 +332,21 @@ func TestErrors(t *testing.T) {
 	out.Reset()
 	if _, err := tpls.Execute(&out, "no_include"); err != nil {
 		errstr := err.Error()
-		if strings.Contains(errstr, "could not be read") {
-			t.Logf("Right error: %s", err.Error())
+		if errors.Is(err, syscall.ENOENT) {
```

Comment:
Not sure if `syscall.ENOENT` isn't a way to specific here, the more general `os.ErrNotExist` should be better suitable, IMHO. I'm not sure about portability of ENOENT (maybe other systems report an other errno, which hopefully will be translated by the Go libraries into `os.ErrNotExist`

Reply:
The _syscall_ package is os dependend.
```
GOOS=windows go doc syscall
GOOS=linux go doc syscall
```
e.g. shows different things. Fortunately for the current situation, `syscall.ENOENT` seems to be available in both cases. But do we know about other OS? 

I'd suggest, using the highest abstraction available, which is `os.ErrNotExist`, IMHO